virttest/utils_misc.py

```diff
@@ -2641,6 +2641,27 @@ def get_free_mem(session, os_type):
     return int(free)
 
 
+def get_used_mem(session, os_type):
+    """
+    Get Used memory for given OS.
+
+    :parm session: shell Object.
+    :parm os_type: os type (eg. linux or windows)
+
+    :return string: usedspace M-bytes
+    """
+    if os_type == "windows":
+        cmd = "systeminfo | findstr /C:'Virtual Memory: In Use'"
+        output = session.cmd_output(cmd)
+        used = "%sM" % re.findall("\d+", output)[0]
+    else:
+        cmd = "free -m | grep 'Mem'"
+        output = session.cmd_output(cmd)
+        used = "%sM" % output.split()[2]
```

Comment:
Need judge output and used is not None.
if output:
    used = ****
     if used:
           float()